File: LC-1486/LC-1486.cpp
class Solution {
public:
    /**
     * Concepts: Naive Approach
     */
    int xorOperation(int n, int start) {
        int result = start, end = start + 2 * n;
        for (int i = start + 2; i < end; i += 2) {
            result ^= i;
        }
        return result;
    }
};

File: LC-1486/LC-1486.js
/**
 * @param {number} n
 * @param {number} start
 * @return {number}
 */
var xorOperation = function (n, start) {
  /**
   * Concepts: Naive Approach
   */

  let result = start,
    end = start + 2 * n
  for (let i = start + 2; i < end; i += 2) {
    result ^= i
  }
  return result
}
